DESCRIPTION

```diff
@@ -90,8 +90,6 @@ Collate:
     'table-formula.R'
     'user-defined-pk-parameter.R'
     'utilities-container.R'
-    'utilities-data-import-configuration.R'
-    'utilities-data-importer-configuration.R'

```

Comment:
@PavelBal I am not sure what these files were about? They were empty or containing an empty function